### ISSUE TITLE
Could not load vanilla model parent 'randomthings:block/platform'

### ISSUE THREAD

**Issue by 0Navis0** [lumien231/Random-Things#98]
I get this error in the log while Minecraft loading, it doesn't provocate any crash, but I don't think that it's normal. http://pastebin.com/2ZhE1KKY

**Comment by 0Navis0** [comment]
Oh I noticed I got that too later in the log http://pastebin.com/3tQHBirD

**Comment by bookerthegeek** [comment]
Still an issue.
 * RandomThings-MC1.10.2-3.7.7.jar
```
[01:55:52] [Client thread/ERROR] [FML]: Could not load vanilla model parent 'randomthings:block/platform' for 'net.minecraft.client.renderer.block.model.ModelBlock@4d52471e
net.minecraftforge.client.model.ModelLoaderRegistry$LoaderException: Exception loading model randomthings:block/platform with loader VanillaLoader.INSTANCE, skipping
	at net.minecraftforge.client.model.ModelLoaderRegistry.getModel(ModelLoaderRegistry.java:153) ~[ModelLoaderRegistry.class:?]
	at net.minecraftforge.client.model.ModelLoaderRegistry.getModelOrLogError(ModelLoaderRegistry.java:203) [ModelLoaderRegistry.class:?]
	at net.minecraftforge.client.model.ModelLoader$VanillaModelWrapper.getTextures(ModelLoader.java:489) [ModelLoader$VanillaModelWrapper.class:?]
	at net.minecraftforge.client.model.ModelLoaderRegistry.getModel(ModelLoaderRegistry.java:163) [ModelLoaderRegistry.class:?]
	at net.minecraftforge.client.model.ModelLoader.func_177590_d(ModelLoader.java:328) [ModelLoader.class:?]
	at net.minecraft.client.renderer.block.model.ModelBakery.func_177577_b(ModelBakery.java:170) [byo.class:?]
	at net.minecraftforge.client.model.ModelLoader.func_177570_a(ModelLoader.java:148) [ModelLoader.class:?]
	at net.minecraft.client.renderer.block.model.ModelManager.func_110549_a(ModelManager.java:28) [byp.class:?]
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110542_a(SimpleReloadableResourceManager.java:122) [bxi.class:?]
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:499) [bcx.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:351) [bcx.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_121]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_121]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:236) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:297) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:162) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:53) [NewLaunch.jar:?]
Caused by: java.io.FileNotFoundException: randomthings:models/block/platform.json
	at net.minecraft.client.resources.FallbackResourceManager.func_110536_a(FallbackResourceManager.java:68) ~[bww.class:?]
	at net.minecraft.client.resources.SimpleReloadableResourceManager.func_110536_a(SimpleReloadableResourceManager.java:65) ~[bxi.class:?]
	at net.minecraft.client.renderer.block.model.ModelBakery.func_177594_c(ModelBakery.java:311) ~[byo.class:?]
	at net.minecraftforge.client.model.ModelLoader.access$1100(ModelLoader.java:118) ~[ModelLoader.class:?]
	at net.minecraftforge.client.model.ModelLoader$VanillaLoader.loadModel(ModelLoader.java:879) ~[ModelLoader$VanillaLoader.class:?]
	at net.minecraftforge.client.model.ModelLoaderRegistry.getModel(ModelLoaderRegistry.java:149) ~[ModelLoaderRegistry.class:?]
	... 25 more
```

**Comment by Yarden-zamir** [comment]
can we get this fixed?

**Comment by lumien231** [comment]
No, unless critical bugs come up there won't be another 1.10.2 update. This is already fixed in 1.11.2.

**Comment by Yarden-zamir** [comment]
Fair enough
